AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -1692,7 +1692,7 @@ delfi.ee#@#.ad-banner
 delfi.ee#@#.patronad
 !#safari_cb_affinity
 !
-duolingo.com#?#body > div[id="root"] > div:not([class]) > div[class^="_"] > div:not([class]) > div[class^="_"] > div[class^="_"] > div[class^="_"]:has(> div[class*=" _"] > h2 +div[class^="_"]:contains(Duolingo Plus))
+duolingo.com#?#body > div[id="root"] > div:not([class]) > div[class^="_"] > div:not([class]) > div[class^="_"] > div[class^="_"]:has(> div[class^="_"] > div[class*=" _"] > h2 +div[class^="_"]:contains(Duolingo Plus))
```

Comment:
Please add one space after the `+` in has part `h2 +div`, so it will be `h2 + div`